### PR TITLE
add: wishlist & cart page

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added Wishlist & Cart pages for e-commerce application.

### CODE REVIEW

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              <small class="sub-heading-light m-b-2">Sold by: Nene Stores</small>
+              <div class="quantity-selector row-flex no-wrap m-b-2">
+                <i class="fa fa-trash-o light-font m-r-2" aria-hidden="true"></i>
+                <i class="fa fa-minus light-font m-r-2" aria-hidden="true"></i>
+                <p class="quantity m-r-2 h3">2</p>
+                <i class="fa fa-plus light-font" aria-hidden="true"></i>
+              </div>
+              <div class="card-footer row-flex full-wd">
+                <p class="h3 colored-text m-r-2 m-b-1">
+                  249₹ <del class="sub-heading-light light-font">499₹ </del>
```
Comment (vrx29) [inline pages/cart.html:75]: Nice you have used 'del' tag as well.

**Review by anik31 — APPROVED** [review]
Nice & clean UI. Code is also good.

**Review by vrx29 — APPROVED** [review]
Looks good to me you can go ahead and merge.

**Comment by nayyyhaa** [comment]
Thanks @vrx29 & @anik31 for reviewing. Merging the PR.

**Pull request merged**